packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -31,6 +31,7 @@ export type SamlAttributes = z.infer<typeof SamlAttributesSchema>;
 
 export const baseRequestSchema = z.object({
   id: z.string(),
+  subRequestId: z.string().optional(),
```

Comment:
Will there always be a subRequestId? If not should we set as optional?

Reply:
nit: I'd use `.nullish()`, since those are used to validate HTTP requests and passing "null" for a prop is a common thing there.